src/frontends/pytorch/src/utils.cpp

```diff
@@ -307,6 +307,16 @@ std::shared_ptr<ov::op::util::FrameworkNode> cast_fw_node(std::shared_ptr<Node>
     return fw_node;
 }
 
+bool is_none_node(std::shared_ptr<Node> node) {
```

Comment:
You use this function on output, so better to change signature to `bool is_none_node(Output<Node> node)`

Reply:
Done